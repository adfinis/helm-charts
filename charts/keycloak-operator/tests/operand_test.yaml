suite: default Keycloak operand tests
templates:
  - keycloak/keycloak.yaml
  - keycloak/keycloakrealmimport.yaml
tests:
  - it: should match snapshot
    values:
      - ../ci/operand-values.yaml
    asserts:
      - matchSnapshot: {}
