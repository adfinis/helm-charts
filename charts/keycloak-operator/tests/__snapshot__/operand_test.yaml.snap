should match snapshot:
  1: |
    apiVersion: k8s.keycloak.org/v2alpha1
    kind: Keycloak
    metadata:
      labels:
        app.kubernetes.io/component: keycloak
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/version: 24.0.4
        helm.sh/chart: keycloak-operator-1.2.0
      name: keycloak
    spec:
      features:
        disabled:
          - js-adapter
          - kerberos
        enabled:
          - account2
          - admin2
          - web-authn
      hostname:
        strict: false
        strictBackchannel: false
      http:
        httpEnabled: true
      ingress:
        enabled: true
      instances: 1
      transaction:
        xaEnabled: false
  2: |
    apiVersion: k8s.keycloak.org/v2alpha1
    kind: KeycloakRealmImport
    metadata:
      labels:
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/version: 24.0.4
        helm.sh/chart: keycloak-operator-1.2.0
      name: RELEASE-NAME-keycloak-operator-test
    spec:
      keycloakCRName: keycloak
      realm:
        id: test
        realm: test
        users:
          - email: john.sausage@example.com
            firstName: Hans
            id: "0"
            lastName: Wurst
            username: john.sausage
